src/pages/guides/eqp/v1/access-keys.md

```diff
@@ -20,8 +20,8 @@ Marketplace EQP API **access keys** are self-serviced.  As long as you are [elig
 ## How do I create an API access key
 
 -  You create your API access key from one, or both, of the **Marketplace Developer Portal** user interfaces:
-   -  production - [https://developer.magento.com][1]
-   -  sandbox - [https://developer-stg.magento.com][2]
+   -  production - [https://commercedeveloper.adobe.com][1]
+   -  sandbox - [https://commercedeveloper-sandbox.adobe.com][2]
 
 ### Create a new API access key
```

Comment:
update Images with the latest UI
For example for step 3, the current experience is the following: 

![image](https://github.com/AdobeDocs/commerce-marketplace/assets/44397551/945d43dc-4d4d-4791-a19a-72e1ac19f236)

Please, update also the images for steps 4, 6 and 7.